layers/+lang/haskell/packages.el

```diff
@@ -189,7 +189,26 @@
     (add-hook 'evil-insert-state-entry-hook 'spacemacs//haskell-indentation-show-guides)
     (add-hook 'evil-emacs-state-entry-hook 'spacemacs//haskell-indentation-show-guides)
     (add-hook 'evil-insert-state-exit-hook 'spacemacs//haskell-indentation-hide-guides)
-    (add-hook 'evil-emacs-state-exit-hook 'spacemacs//haskell-indentation-hide-guides)))
+    (add-hook 'evil-emacs-state-exit-hook 'spacemacs//haskell-indentation-hide-guides))
+
+  ;; align rules for Haskell
+  (with-eval-after-load 'align
+    (add-to-list 'align-rules-list
+                 '(haskell-types
+                   (regexp . "\\(\\s-+\\)\\(::\\|∷\\)\\s-+")
+                   (modes quote (haskell-mode literate-haskell-mode))))
```

Comment:
Why `quote` ?
Can you replace them with `(modes . '(haskell-mode literate-haskell-mode))` ?